staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -702,6 +752,7 @@ func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 
 		if plState.queues == nil {
 			klog.V(5).Infof("Introducing queues for priority level %q: config=%s, concurrencyLimit=%d, quiescing=%v (shares=%v, shareSum=%v)", plName, fcfmt.Fmt(plState.pl.Spec), concurrencyLimit, plState.quiescing, plState.pl.Spec.Limited.NominalConcurrencyShares, meal.shareSum)
+			plState.seatDemandStats = newSeatDemandStats(float64(concurrencyLimit))
```

Comment:
Yes, I had those thoughts while writing the KEP update.  What I landed on for the KEP is:

>  For a newly introduced priority level, we set HighSeatDemand, AvgSeatDemand, and SmoothSeatDemand to NominalCL-LendableSD/2 and StDevSeatDemand to zero.

I can't do exactly that in this PR, because we do not have LendableSD values yet.  I will add a forward-looking comment.